tests/testMakePixelToTanPixel.py

```diff
@@ -26,14 +26,73 @@
 """
 import unittest
 
-import numpy
+import math
 
 import lsst.utils.tests
 import lsst.afw.geom as afwGeom
 import lsst.afw.cameraGeom as cameraGeom
 from lsst.afw.cameraGeom import makePixelToTanPixel
 
-class MakePixelToTanPixelTestCaseCase(unittest.TestCase):
+class MakePixelToTanPixelTestCaseCase(lsst.utils.tests.TestCase):
+    def testSimpleCurvedFocalPlane(self):
```

Comment:
I feel it is important because it allows some low-level sanity checks that are difficult with a more complex geometry (e.g. a detector with rectangular pixels and offset and rotated with respect to the focal plane, as is used in testCurvedFocalPlane). In particular I can check that correction is purely radial. I agree that the comparison of actual pixel positions is redundant, but I felt it best to leave it in as an extra check.